src/OrchardCore.Build/Dependencies.AspNetCore.props

```diff
@@ -18,4 +38,16 @@
     <PackageManagement Include="Microsoft.Extensions.Caching.StackExchangeRedis" Version="$(AspNetCoreVersion)" />
     <PackageManagement Include="Microsoft.Extensions.Http.Polly" Version="$(AspNetCoreVersion)" />
   </ItemGroup>
+
+  <!-- For .NET 5.0 packages that do not have a version number that matches $(AspNetCoreVersion) available -->
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net5.0' ">
```

Comment:
Create a `<MicrosoftExtensionsVersion>5.0.1</MicrosoftExtensionsVersion>` tag like AspNetCoreVersion instead.

Reply:
Great call, I'll do that.

Reply:
@sebastienros Done